stable/lamp/examples/drupal.yaml

```diff
@@ -5,8 +5,7 @@ mysql:
   database: drupal
 
 php:
-  repository: drupal
-  tag: 7-fpm
+  image: "drupal:fpm-alpine"
```

Comment:
this would be a breaking change and would require semver bumped a full version (or at least a minor since this is a not even version 1 ).